core/pom.xml

```diff
@@ -23,7 +23,7 @@
   <parent>
     <groupId>com.dibs</groupId>
     <artifactId>dozer-parent</artifactId>
-    <version>5.5.2-PDFIX</version>
```

Comment:
Not really sure what we want here for the version.

Reply:
I think we can probably just rev the patch version so 5.5.3-PDFIX

Reply:
My only concern with this is that dozer uses its patch versions so it would make it seem like we forked off of dozer's 5.5.3 and did a PDFIX on that.

Reply:
ahh good point, add a patch version after PDFIX? 5.5.2-PDFIX.1?